helm_deploy/court-case-service/templates/_envs.tpl

```diff
@@ -55,9 +55,24 @@ env:
   - name: DELIUS_BASE_URL
     value: "{{ .Values.env.delius.base_url }}"
 
+  - name: COURT_CASE_AND_DELIUS_BASE_URL
+    value: "{{ .Values.env.court_case_and_delius_api.base_url }}"
+
   - name: MANAGE_OFFENCES_API_BASE_URL
     value: "{{ .Values.env.manage_offences_api.base_url }}"
 
+  - name: COURT_CASE_AND_DELIUS_API_CLIENT_ID
```

Comment:
Can we not reuse OFFENDER_ASSESSMENTS_API_CLIENT_ID as it fetches same credentials?